src/frontend/src/pages/GanttPage/GanttChartTeamSection.tsx

```diff
@@ -77,12 +91,52 @@ const GanttChartTeamSection = ({
           height: '30px'
         }}
       >
+        <AddProjectModal
+          showModal={showAddProjectModal}
+          handleClose={() => setShowAddProjectModal(false)}
+          addProject={(project) => {
+            const newProject: GanttTask = {
+              id: id + projectTasks.length + 1,
+              name: project.name,
+              start: new Date(),
+              carNumber: project.carNumber,
+              projectNumber: getNewProjectNumber(project.carNumber),
+              end: new Date(Date.now() + 1000 * 60 * 60 * 24 * 30),
+              workPackages: [],
+              teamName,
+              type: 'project'
+            };
+
+            addProject(newProject);
+
+            projectTasks.push(newProject);
+
+            handleEdit();
+
+            createChange({
+              id,
+              eventId: newProject.id,
+              type: 'create-project'
+            });
+          }}
```

Comment:
same comment as that other inline, this is pretty complex if we could move it above the return and give it a good name/ a high level comment, that'd be good